From 6f3a20c731dbde18a433a1a1502c006b87a6acb4 Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Mon, 9 Jul 2018 18:56:25 -0400
Subject: [PATCH] joyent/node-mooremachine#17 package.json repository does not
 point to Joyent Reviewed by: Alex Wilson <alex.wilson@joyent.com> Approved
 by: Alex Wilson <alex.wilson@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index a3b1ac2..379f99a 100644
--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/arekinath/node-mooremachine.git"
+    "url": "git+https://github.com/joyent/node-mooremachine.git"
   },
   "author": "Joyent, Inc",
   "contributors": [
-- 
2.21.0

